ouroboros-consensus-cardano/test/cardano-test/Test/ThreadNet/Cardano.hs

```diff
@@ -478,31 +478,14 @@ mkProtocolCardanoAndHardForkTxs
                                           -- window so that the forks induced by
                                           -- the network partition are as deep
                                           -- as possible.
+        (SL.ProtVer conwayMajorVersion  0) -- During the tests, we want to hard fork all the way to Conway, so we need to signal that we support that era. This version will have to be changed if this test has to hard fork to eras beyond Conway.
```

Comment:
Long line

Reply:
I wanted to avoid introducing a lot of newlines. But I think it's better to be consistent with the style above this line :+1: